src/routes/cleanAll.ts

```diff
@@ -1,9 +1,15 @@
 import { RequestHandler } from 'express'
 import { Queue as QueueMq } from 'bullmq'
 
-import { BullBoardQueues } from '../@types'
+import { BullBoardQueues } from '../@types/app'
 
-export const cleanAll: RequestHandler = async (req, res) => {
+type RequestParams = {
+  queueName: string
+  // REVIEW: please confirm we can't receive 'paused' nor 'latest' here
```

Comment:
cc @vcapretz

Reply:
nope, this is currently not implemented, you can only clean delayed and failed queues for now 😄